english/explore-outside/talk-child/talk-child.md

```diff
@@ -9,3 +9,5 @@ Do you:
 the nearest exit out of the Matrix.](../journey-to-flaming-tower/exit/exit.md)
 
 [Try a MacGyver escape move with items in pocket.](../../macgyver/macgyver.md)
+
+[Flash the sweetest smile to the child?](../../smile/smile-to-child.md)
```

Comment:
Looks like this link is still broken. I believe will will only need a single `..` since you're only moving up one directory.